gufe/network.py

```diff
@@ -11,14 +11,19 @@
 
 
 class AlchemicalNetwork(GufeTokenizable):
-    """A network of `ChemicalSystem`s as nodes, `Transformation`s as edges.
+    """A network with all the information needed for a simulation campaign.
+
+    Nodes are :class:`.ChemicalSystem`\ s and edges are
+    :class:`.Transformation`\ s.
 
     Attributes
     ----------
     edges : FrozenSet[Transformation]
-        The edges of the network, given as a `frozenset` of `Transformation`s.
+        The edges of the network, given as a ``frozenset`` of
+        :class:`.Transformation`\ s.
```

Comment:
I think somewhere there's a compromise between it being rendered nicely in docs and the notebook inspection docs being readable, and we're losing here.  This seems like the notebook docs got a lot worse for a link to `Transformation` that was already given above.  I don't think every single mention of a class needs to be a cross reference.